src/regtech_api_commons/oauth2/oauth2_admin.py

```diff
@@ -8,7 +8,7 @@
 
 from keycloak import KeycloakAdmin, KeycloakOpenIDConnection, exceptions as kce
 
-from regtech_api_commons.models import RegTechUser
+from ..models.auth import RegTechUser
 
```

Comment:
According to a python standard (had to look this up) "Relative imports for intra-package imports are highly discouraged. Always use the absolute package path for all imports. "

So I think for this and the file below, we should switch the relative imports to be 'regtech_api_commons.models.auth' and do away with '..' notation.  Assuming that works.

Reply:
Ooops, missed that.